### PR TITLE
feat(app-context): add app context

### PR DESCRIPTION
**Pull request by DarkNebula0**
Update packages and add application context

### CODE REVIEW

File: pkg/app/package.json [hunk 1]
```diff
@@ -13,11 +13,12 @@
     "@emotion/react": "^11.5.0",
     "@emotion/server": "^11.4.0",
     "@emotion/styled": "^11.3.0",
+    "@gamedao-haiku/translations": "*",
```
Comment (2075) [inline pkg/app/package.json:16]: we also have a gamedao group on npm

File: pkg/app/src/@types/app.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+export enum AppActionTypes {
+	CONFIG_LOAD = 'CONFIG_LOAD',
+	CONFIG_UPDATE = 'CONFIG_UPDATE',
+	FEATURES_LOAD = 'FEATURES_LOAD',
+	FEATURES_UPDATE = 'FEATURES_UPDATE',
+}
+
+export interface AppState {
```
Comment (2075) [inline pkg/app/src/@types/app.ts:19]: why interface for a struct vs types? interfaces make only sense if we represent the data structures as classes (e.g. for the gql) — but this opinion could be outdated as well.

File: pkg/app/src/provider/app/modules/app.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { AppAction, AppActionTypes } from 'src/@types/app'
+import type { Dispatch } from 'react'
+
+// TODO: get root config through .env
+const HOST = 'api/'
+const ENV = 'dev'
+
+async function loadAppDat(resource: string, env: string) {
```
Comment (2075) [inline pkg/app/src/provider/app/modules/app.ts:8]: please add an 'a', but will be replaced with a graphql query at some point

**Review by 2075 — APPROVED** [review]

File: pkg/app/src/@types/app.ts (hunk 2 above)
Comment (DarkNebula0) [reply pkg/app/src/@types/app.ts:19]: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Is basically the same the advantage of interface is that you can extend it

File: pkg/app/src/provider/app/modules/app.ts (hunk 3 above)
Comment (DarkNebula0) [reply pkg/app/src/provider/app/modules/app.ts:8]: 🥇

File: pkg/app/src/@types/app.ts (hunk 2 above)
Comment (2075) [reply pkg/app/src/@types/app.ts:19]: yes, and types can be composed as well :)

**Review by 2075 — DISMISSED** [review]

**Review by 2075 — APPROVED** [review]

**Pull request merged**